pkg/blobfuse-proxy/install-proxy-rhcos.sh

```diff
@@ -17,10 +17,25 @@
 set -xe
 
 # in coreos, we could just copy the blobfuse2 binary to /usr/local/bin/blobfuse2
+updateBlobfuse2="true"
```

Comment:
updateBlobfuse2 should be true only when INSTALL_BLOBFUSE2 is true

Reply:
It will be set to false when INSTALL_BLOBFUSE2 is not true in line 33.

Reply:
updateBlobfuse2 will be set as true only when INSTALL_BLOBFUSE2 is true and blobfuse2 does not exist or the old binary file is not the same as the new file.